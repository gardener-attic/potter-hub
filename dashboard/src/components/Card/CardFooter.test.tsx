import { shallow } from "enzyme";
import * as React from "react";

import CardFooter from "./CardFooter";

it("should render the className", () => {
  const wrapper = shallow(<CardFooter className="foo" />);
  expect(wrapper.find(".foo")).toExist();
});

it("should render the children elements", () => {
  const wrapper = shallow(
    <CardFooter>
      <div>foo</div>
    </CardFooter>,
  );
  expect(wrapper.text()).toContain("foo");
  expect(wrapper).toMatchSnapshot();
});
