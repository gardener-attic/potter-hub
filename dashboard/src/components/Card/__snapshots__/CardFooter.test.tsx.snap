// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the children elements 1`] = `
<div
  className="Card__footer padding-h-normal "
>
  <hr
    className="separator-small"
  />
  <div
    className="padding-b-normal"
  >
    <div>
      foo
    </div>
  </div>
</div>
`;
